tardis/conftest.py

```diff
@@ -133,10 +133,10 @@ def pytest_report_header(config):
     if opts:
         s += "Using Astropy options: {0}.\n".format(" ".join(opts))
 
-    if six.PY3 and (config.getini('doctest_rst') or config.option.doctest_rst):
+    if not six.PY2 and (config.getini('doctest_rst') or config.option.doctest_rst):
```

Comment:
okay - I think we should leave it as the old one. The only reason is that when our astropy friends will help us, they are not confused. please change it back.